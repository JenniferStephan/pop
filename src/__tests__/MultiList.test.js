import React from "react";
import MultiList from "../MultiList";
import renderer from "react-test-renderer";

it("renders correctly", () => {
  const tree = renderer.create(<MultiList />).toJSON();
  expect(tree).toMatchSnapshot();
});

it("renders correctly with more options", () => {
  const tree = renderer
    .create(
      <MultiList
        dataField="CONTIENT_IMAGE.keyword"
        title="Contient une image"
        componentId="image"
        placeholder="oui ou non"
        showSearch={false}
        defaultSelected={[]}
        onCollapseChange={() => {}}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
