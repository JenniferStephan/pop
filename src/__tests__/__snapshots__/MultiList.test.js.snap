// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="multilist"
  style={Object {}}
>
  <div
    className="topBar"
    onClick={[Function]}
  >
    <div
      className="name"
    />
    <div
      className="v"
    >
      ⌄
    </div>
  </div>
  <div
    className="collapse"
    style={Object {}}
  />
</div>
`;

exports[`renders correctly with more options 1`] = `
<div
  className="multilist"
  style={Object {}}
>
  <div
    className="topBar"
    onClick={[Function]}
  >
    <div
      className="name"
    >
      Contient une image
    </div>
    <div
      className="v"
    >
      ⌄
    </div>
  </div>
  <div
    className="collapse"
    style={Object {}}
  />
</div>
`;
